pom.xml

```diff
@@ -172,6 +178,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency> <!-- optional dep of workflow-scm-step, have to override 1.54 version that gets pulled in somehow during tests -->
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>subversion</artifactId>
+            <version>2.5</version>
```

Comment:
`workflow-scm-step` does not _advocate_ use of 2.5. Whatever the current release is on the update center, that is what the user will get. It simply _requires_ 2.5 or higher.